kubernetes/roles/deploy-kubevirt/tasks/main.yml

```diff
@@ -1,14 +1,67 @@
-- name: "List all manifests"
+---
+- include_role:
+    name: ansible.kubernetes-modules
+  when: wait_kubevirt_rollout
+
+- file:
+    path: "{{ manifests_path }}"
+    recurse: True
+    state: directory
+
+- name: git clone kubevirt master
```

Comment:
I think that we should add an option to consume the published `kubevirt.yaml` (https://github.com/kubevirt/kubevirt/releases/download/v0.1.0/kubevirt.yaml), this way we will verify that it was created correctly. 
If there is an unstable `kubevirt.yaml` that is being generated every time the kubevirt containers are being built, it will be the best.

Reply:
@gbenhaim is it not the same file that is available in `manifests/release`?
I certainly can make it an option in any case.